### ISSUE TITLE
GLib Warning: GError set over the top of a previous GError

### ISSUE THREAD

**Issue by gerickson** [Nuovations/connman#9]
When launching `connmand` with:

```
# CONNMAN_RESOLV_DEBUG=1 CONNMAN_WEB_DEBUG=1 /usr/sbin/connmand -n -d -W nl80211
```

GLib reports the following error:

```
connmand[538]: Connection Manager version 1.42
connmand[538]: src/dbus.c:__connman_dbus_init() 
connmand[538]: src/main.c:parse_config() parsing main.conf

(connmand:538): GLib-WARNING **: 10:34:04.783: GError set over the top of a previous GError or uninitialized memory.
This indicates a bug in someone's code. You must ensure an error is NULL before it's set.
The overwriting error message was: Key file does not have key ?ResolvConf? in group ?General?
```

**Comment by gerickson** [comment]
This seems to be related to #4.

**Comment by gerickson** [comment]
Closed with #19.
